modules/gittar/helper/git_cmd.go

```diff
@@ -183,10 +183,15 @@ func RunProcess(service string, c *webcontext.Context) {
 		}
 		logrus.Infof("push header:" + string(header))
 		re := regexp.MustCompile(
-			`(?mi)(?P<before>[0-9a-fA-F]{40}) (?P<after>[0-9a-fA-F]{40}) (?P<ref>refs\/(heads|tags)\/.*?)\0`,
+			`(?mi)(?P<before>[0-9a-fA-F]{40}) (?P<after>[0-9a-fA-F]{40}) (?P<ref>refs\/(heads|tags)\/\S*)`,
 		)
+		// remove 0000
```

Comment:
you should add some comment or link to explain why remove 0000

and add unit test if could.

Reply:
done